services/lambda/src/it/java/software/amazon/awssdk/services/lambda/ServiceIntegrationTest.java

```diff
@@ -81,6 +84,9 @@ public void uploadFunction() throws IOException {
                                                                     .timeout(10)
                                                                     .role(lambdaServiceRoleArn)).join();
 
+        lambda.waiter()
```

Comment:
I wonder if we should make it a static variable and use `@BeforeClass` instead because creating a function for each individual test can prolong the test time

Reply:
done